Makefile

```diff
@@ -67,3 +67,11 @@ clean:
 	find . -name '__pycache__' -delete
 	rm -rf .tox
 	rm -rf .paasta
+
+yelpy: ## Installs the yelp-internal packages into the default tox environment
+	.tox/py36-linux/bin/pip-custom-platform install -i https://pypi.yelpcorp.com/simple -r yelp_package/extra_requirements_yelp.txt -r ./extra-linux-requirements.txt
```

Comment:
This may make more sense as a tox environment, but I like where you're going. This shouldn't just be something that I have to search for in bash history.